src/udev/udevadm-control.c

```diff
@@ -41,6 +41,16 @@ static int arg_start_exec_queue = -1;
 
 STATIC_DESTRUCTOR_REGISTER(arg_env, strv_freep);
 
+static int send_reload(UdevConnection *conn) {
```

Comment:
Even this is used by udevadm-control.c, I'd like to move this to udev-connection.c. Though, not a big matter.